src/Auth/Contracts/AuthTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Auth\Contracts;
+
+use Kanvas\Auth\Factory;
+use Kanvas\Auth\TokenResponse;
+use Kanvas\Users\Users\Models\Users;
+use Kanvas\Auth\Contracts\TokenTrait;
+use Illuminate\Http\Request;
+
+trait AuthTrait
```

Comment:
@rwhite27 traits now go in the folder traits , contracts are just for interface , this was a mistake on 0.3

Reply:
Understood.